smithy-diff/src/main/java/software/amazon/smithy/diff/evaluators/AddedShape.java

```diff
@@ -38,4 +45,27 @@ private boolean isMemberOfAddedShape(Shape shape, Differences differences) {
                 .filter(member -> !differences.getOldModel().getShapeIds().contains(member.getContainer()))
                 .isPresent();
     }
+
+    private boolean isMemberOfConvertedStringWithEnumTraitToEnumShape(Shape shape, Differences differences) {
```

Comment:
I think having it explicit makes it so that there is no confusion about which way the conversion is happening:

- Support `StringShape` with `@enum` trait -> `EnumShape`
- DON'T support `EnumShape` -> `StringShape` with `@enum` trait